servers/cromwell/jobs/test/test_task_statuses.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+
+from . import BaseTestCase
+
+from jobs.controllers.utils import task_statuses
+
+
+class TestTaskStatuses(BaseTestCase):
+    # yapf: disable
+    def test_cromwell_execution_to_api_maps_all_execution_statuses_correctly(self):
```

Comment:
This test seems unnecessary to me since it's just testing a dictionary look-up in `CROMWELL_EXECUTION_STATUS_MAP`